spec/unit/defaults_spec.lua

```diff
@@ -0,0 +1,90 @@
+describe("defaults module", function()
+    local Defaults, DataStorage
+    setup(function()
+        require("commonrequire")
+        Defaults = require("apps/filemanager/filemanagersetdefaults")
+        DataStorage = require("datastorage")
+    end)
+
+    it("should load all defaults from defaults.lua", function()
+        Defaults:init()
+        assert.is_same(#Defaults.defaults_name, 78)
+        assert.is_same(Defaults.defaults_name[29], 'DHINTCOUNT')
```

Comment:
Yeah, the original code structure is not easy to test. I ended up doing this because I am thinking we will not be adding any new default globals in the future. Eventually, this whole defaults thing should go away in favor of proper setting files. So yes, we still need to update the test case when we remove defaults. Let me know if you came up with better way to test this.